src/BitmapSkin.js

```diff
@@ -122,8 +122,8 @@ class BitmapSkin extends Skin {
      * @param {twgl.v3} vec A texture coordinate.
      * @return {boolean} Did it touch?
      */
-    isTouching (vec) {
-        return this._silhouette.isTouching(vec);
+    isTouching (vec, interpolate) {
```

Comment:
I don't know about this API ...

At minimum, update the jsdoc block.

Reply:
yeah - maybe we should always interp?

Reply:
Uh. I'm not sure about the default, interpolation is fine as the default.

I do think we should have two separate `isTouching` methods with descriptive names whether they use nearest neighbor or interpolation. We could keep `isTouching` as a third method but it would only have the current one argument.